docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Why are you using the <code>rstac::</code> prefix in this cell when you're not using it an any of the previous cells? Again, be consistent. Either <em>always</em> use the prefix or <em>never</em> use the prefix.</p><p> </p><p>Further, you defined <code>stac_endpoint</code> at the top of the notebook and use it for earlier queries, so use it here as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>